File: Habitica/src/main/java/com/habitrpg/android/habitica/widget/DailiesListFactory.java
package com.habitrpg.android.habitica.widget;

import android.content.Context;
import android.content.Intent;

import com.habitrpg.android.habitica.R;
import com.magicmicky.habitrpgwrapper.lib.models.tasks.Task;


public class DailiesListFactory extends TaskListFactory {
    public DailiesListFactory(Context context, Intent intent) {
        super(context, intent, Task.TYPE_DAILY, R.layout.widget_dailies_list_row, R.id.dailies_text);
    }
}

File: Habitica/src/main/java/com/habitrpg/android/habitica/widget/DailiesListFactory.kt
package com.habitrpg.android.habitica.widget

import android.content.Context
import android.content.Intent
import com.habitrpg.android.habitica.R
import com.habitrpg.android.habitica.data.TaskRepository
import com.habitrpg.android.habitica.data.UserRepository
import com.habitrpg.shared.habitica.models.tasks.TaskType

class DailiesListFactory(context : Context, intent : Intent, taskRepository : TaskRepository, userRepository : UserRepository) : TaskListFactory(
    context,
    intent,
    TaskType.DAILY,
    R.layout.widget_dailies_list_row,
    R.id.dailies_text,
    taskRepository,
    userRepository
)
